### PR TITLE
Adding AlmaLinux

### PR DESCRIPTION
**Pull request by bennyvasquez**
I'm hoping I did things right here! Happy to take feedback if I need to adjust anything.

### CODE REVIEW

File: _foundations/almalinux.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+boardurl: https://wiki.almalinux.org/Transparency.html
+bylawsurl: https://almalinux.org/p/foundation-bylaws/
+numberOfEmployees: 0
+funding: sponsors
+budgetUsd: 
+budgetYear:
+budgeturl: 
+sponsorurl: https://almalinux.org
+sponsorList: Hawk Host, OpenLogic, CERN, black.host, procomputers.com, Hivelocity, Mattermost, Cybertrust Japan Co., Codenotary, Atix, World4You Internet Services GmbH, CloudLinux Inc, KnownHost, LLC, WebPros, Inc (under cPanel and Plesk brands), IPInfo
+licenses: https://www.osgeo.org/about/licenses/
```
Comment (ShaneCurcuru) [inline _foundations/almalinux.md:24]: Is this really the correct license link? 😸

File: _foundations/almalinux.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+numberOfEmployees: 0
+funding: sponsors
+budgetUsd: 
+budgetYear:
+budgeturl: 
+sponsorurl: https://almalinux.org
+sponsorList: Hawk Host, OpenLogic, CERN, black.host, procomputers.com, Hivelocity, Mattermost, Cybertrust Japan Co., Codenotary, Atix, World4You Internet Services GmbH, CloudLinux Inc, KnownHost, LLC, WebPros, Inc (under cPanel and Plesk brands), IPInfo
+licenses: https://www.osgeo.org/about/licenses/ 
+claPolicy: 
+ethicsPolicy:https://almalinux.org/p/foundation-board-operations-and-ethics/
```
Comment (ShaneCurcuru) [inline _foundations/almalinux.md:26]: Typo, needs a space after the first colon (we still need to add linting to commit checks)

**Review by ShaneCurcuru — CHANGES_REQUESTED** [review]
Thanks - a couple of errors to fix first.  Also, do you want to add links to your brandPolicy, and any code of conduct you have?

**Comment by bennyvasquez** [comment]
@ShaneCurcuru Good catch on both of those! Can you tell I was finding examples in other listings to help? :D This push should fix them both, I think.

**Pull request merged**